control/matlab.py

```diff
@@ -1525,17 +1518,12 @@ def tfdata(sys, **kw):
     sys: Lti (StateSpace, or TransferFunction)
         LTI system whose data will be returned
 
-    Keywords
-    --------
-    inputs = int; outputs = int
-        For MIMO transfer function, return num, den for given inputs, outputs
-
     Returns
     -------
     (num, den): numerator and denominator arrays
         Transfer function coefficients (SISO only)
     '''
-    tf = _convertToTransferFunction(sys, **kw)
```

Comment:
The keyword arguments `inputs` and `outputs` to `_convertToTransferFunction` apply only if `sys` is a SISO system, and the result is to return a MIMO transfer function with the specified number of inputs and outputs.  So this is different from what is in the documentation string above, and since this routine just extracts the numerator and denominator of a transfer function, there would never be a need to convert a SISO system to MIMO.